observability/micrometer/src/main/java/org/wildfly/extension/micrometer/MicrometerSubsystemSchema.java

```diff
@@ -32,8 +33,10 @@ public VersionedNamespace<IntVersion, MicrometerSubsystemSchema> getNamespace()
 
     @Override
     public PersistentResourceXMLDescription getXMLDescription() {
-        return builder(org.wildfly.extension.micrometer.MicrometerExtension.SUBSYSTEM_PATH, this.namespace)
-                .addAttributes(MicrometerSubsystemDefinition.ATTRIBUTES)
-                .build();
+        PersistentResourceXMLDescription.PersistentResourceXMLBuilder builder =
+                builder(MicrometerSubsystemRegistrar.PATH, this.namespace);
```

Comment:
As long as you're touching this now, perhaps update so not to rely on deprecated API?

Reply:
Definitely. I think I looked at that code and all it said was "deprecated" with no alternative offered. Point me to it?

Reply:
There should be @deprecated javadoc tags:
https://github.com/wildfly/wildfly-core/blob/main/controller/src/main/java/org/jboss/as/controller/PersistentResourceXMLDescription.java#L541

Reply:
Ah, ok. That wasn't popping up in IDEA, but I'll open that source directly. Thanks.